app/models/requests/request.rb

```diff
@@ -263,6 +263,8 @@ def status_label(item:, availability_data:)
           "Not Available"
         elsif item["status_source"] != "work_order" && item[:status_label] == 'Item in place' && availability_data.size == 1 && availability_data.first['errorMessage'] == "Bib Id doesn't exist in SCSB database."
           "In Process"
+        elsif item["status_source"] == "work_order" && item["status"] == "Not Available" && item[:status_label] != "Acquisitions and Cataloging"
```

Comment:
@maxkadel why 'unless in acquisitions it's not available' ? Where is this information coming from? I dont see it in the ticket.

Reply:
This was my understanding of the conversation in the Orangelight/Bibdata meeting last week, but I may have misunderstood.